pkg/volume/gce_pd/gce_util.go

```diff
@@ -194,7 +196,7 @@ func (gceutil *GCEDiskUtil) CreateVolume(c *gcePersistentDiskProvisioner) (strin
 		glog.Errorf("error getting labels for volume %q: %v", name, err)
 	}
 
-	return name, int(requestGB), labels, fstype, nil
+	return name, int(requestGiB), labels, fstype, nil
```

Comment:
Lets make sure that value returned by this function is read correctly by `Provision` function here - https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/gce_pd/gce_pd.go#L426 . Currently as it exists it will try to save a GB value back into GiB that will be wrong.

Reply:
Ah, I missed that you are creating both GB and GiB values out of requested size and returning GiB value back to caller. That works too I think.

Reply:
@gnufied Thanks your review.

Reply:
Would be worth renaming the variables GB here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/gce_pd/gce_pd.go#L405
and here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/gce_pd/gce_pd.go#L426

Reply:
@davidz627 Done, thanks.

Reply:
I'm confused, why do we need to return GiB if we're allocating in GB?  Won't the size recorded in the PV be wrong then?

Reply:
Hi @msau42, Thanks your review. Does your suggestion is to remove [this line ](https://github.com/kubernetes/kubernetes/pull/56600/files#diff-6f726717375311cc43c305d7a65cf2a9R86) and return requestGB directly,  and replace "%dGi" with "%dG" in [this line](https://github.com/kubernetes/kubernetes/pull/56600/files#diff-f7811184a41099ebc39b015ff49f491bR426)?

Reply:
I think either thing works. Return the GB value and then get `resource.Quantity` out of it by doing `%dG`. That is what I originally suggested above.  Returning `GiB` and leaving `%dGi` works too, but I do think that may be returning GB and then parsing value back using `%dG` was better.

Reply:
I don't think returning GiB is accurate.  requestGiB != requestGB in bytes.  The PV is going to have capacity larger than what was actually allocated.

So if there is no reason to return values in Gi, then I would recommend just returning GB.

Reply:
@msau42 @gnufied Done. Now it is return GB Directly. Thanks.